tests/python/gaia-ui-tests/gaiatest/form_controls/header.py

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from abc import ABCMeta, abstractproperty
+from marionette_driver import By, Wait, expected
+from gaiatest.form_controls.form_control import Widget
+
+
+class GaiaHeader(Widget):
+    # This represents the gaia-header element
+
+    __metaclass__ = ABCMeta
+
+    def go_back(self):
+        Wait(self.marionette).until(expected.element_enabled(self.root_element) and
+                                    expected.element_displayed(self.root_element))
+        # TODO: remove tap with coordinates after Bug 1061698 is fixed
+        self.root_element.tap(x=20)
+        Wait(self.marionette).until(expected.element_not_displayed(self.root_element))
+
+    def go_back_and_exit(self, app=None, add_statusbar_height=True):
+        self.tap_element_from_system_app(self.root_element, add_statusbar_height=add_statusbar_height, x=20)
```

Comment:
Why don't we add `add_statusbar_height` to `go_back()`?

I think `go_back_and_exit()` has to call `go_back()` because we don't perform the waits in this methods.